stats/stats.go

```diff
@@ -104,6 +106,9 @@ type InTrailer struct {
 	Client bool
 	// WireLength is the wire length of trailer.
 	WireLength int
+	// Trailer contains the trailer metadata received from the server. This
+	// field is only valid if this InTrailer is from the client side.
+	Trailer metadata.MD
```

Comment:
Please add `// Deprecated: use Trailer in InTrailer instead.` to the comment on `End.Trailer`.

This package is explicitly marked as experimental, so we can eventually remove the other usage after allowing sufficient time for migration to the new field.

Reply:
Sounds good. It would be nice to simplify the duplication. I added the comment below.